src/main/scala/com/databricks/spark/sql/perf/mllib/feature/StringIndexerModel.scala

```diff
@@ -0,0 +1,39 @@
+package com.databricks.spark.sql.perf.mllib.feature
+
+import org.apache.spark.ml
+import org.apache.spark.ml.linalg.Vector
+import org.apache.spark.ml.PipelineStage
+import org.apache.spark.sql._
+
+import com.databricks.spark.sql.perf.mllib.OptionImplicits._
+import com.databricks.spark.sql.perf.mllib.data.DataGenerator
+import com.databricks.spark.sql.perf.mllib.{BenchmarkAlgorithm, MLBenchContext, TestFromTraining}
+
+/** Object for testing StringIndexer performance */
+object StringIndexerModel extends BenchmarkAlgorithm with TestFromTraining with UnaryTransformer {
+
+  val distinctCount = 10000
```

Comment:
Same comment as above, maybe distinctCount should be a user-specified param in `MLParams`